docs/REST_API_V1.md

```diff
@@ -1,3 +1,6 @@
+# Notice
+This document describes API v1, for info about API v2 see [REST_API_V2.md](REST_API_V2.md).
```

Comment:
This document describes API v1 which is stable still supported, but no longer in active development. 
In the future, it will be fully replaced by [REST_API_V2.md](REST_API_V2.md) - see details in [the issue 1637](https://github.com/intelsdi-x/snap/issues/1637). 
The process of depreciation will be preceded by an announcement and giving time to switch Snap users to REST API V2. 

@kdembler, could you provide the table with information what changes in REST API V2?